src/web/SystemAO/locale/personalization.json

```diff
@@ -164,6 +164,88 @@
             },
             "placeholder":{
 
+            }
+        },
+        "zh-cn": {
+            "fwtitle" : "个人化",
+            "fontFamily":"\"Microsoft JhengHei\",\"SimHei\", \"Apple LiGothic Medium\", \"STHeiti\"",
```

Comment:
Hi @pi-dal , your commits looks great to me :D
I notice you are using the same font sets that is used by Traditional Chinese localization as I highlighted here. You sure these font sets can also be used to show Simplified Chinese characters as well?